### ISSUE TITLE
Tenshi not working

### ISSUE THREAD

**Issue by CoLDHearT18** [justfoolingaround/animdl#181]
**Describe the bug**
When using "tenshi", the stream starts -> mpv opens -> then immediately mpv closes, showing "detected a non zero return code: 2" error. "Animixplay and Animepahe" are working fine.

**Did the error occur in between an active task (`download` / `stream` / `grab`)?**
Error occurs only with "download and stream", grab works fine.

**Screenshots [Optional]**
![Screenshot (68)](https://user-images.githubusercontent.com/113617419/191595289-c2e28524-caff-43d9-9851-5fc0a18e0c1c.png)

**Additional information**
- animdl version 1.6.21 (Idk if this is the latest version, I dont know how to update)

**Comment by onomatopellan** [comment]
They changed something that makes the best quality option to fail (timeline.vtt). But the other ones still work.

```
$ animdl stream "tenshi:death note" -r 1 -q auto
                                                justfoolingaround/animdl - v1.6.21
                                       A highly efficient, powerful and fast anime scraper.
[prompts.py:40 - 2022-09-22 02:55:57,968 - streamer: INFO] 01: Death Note / https://tenshi.moe/anime/izt6uppv
Select the search result using index [1]:
[stream.py:68 - 2022-09-22 02:55:59,319 - tenshi/streamer: INFO] Now initiating your stream session
[prompts.py:176 - 2022-09-22 02:56:00,947 - tenshi/streamer: INFO] Uncategorised
[prompts.py:178 - 2022-09-22 02:56:00,947 - tenshi/streamer: INFO] 720
[prompts.py:180 - 2022-09-22 02:56:00,947 - tenshi/streamer: INFO] Hard Subtitles
[prompts.py:182 - 2022-09-22 02:56:00,947 - tenshi/streamer: INFO] 01 / timeline.vtt / tenshi.moe
[prompts.py:178 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] 480
[prompts.py:180 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] Hard Subtitles
[prompts.py:182 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] 02 / 89c38d0829a0279d / seiryuu.tenshi.moe
[prompts.py:178 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] 360
[prompts.py:180 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] Hard Subtitles
[prompts.py:182 - 2022-09-22 02:56:00,948 - tenshi/streamer: INFO] 03 / 855fe44bf4164d87 / seiryuu.tenshi.moe
Select above, using the stream index [1]: 2
[stream.py:118 - 2022-09-22 02:56:07,575 - tenshi/streamer: INFO] Streaming 'Death Note: Episode 1', [1/1, 0 remaining]
```

**Comment by CoLDHearT18** [comment]
@onomatopellan yeah I just checked, 720 and 1080 dont work, only 480 works. Hoping for a fix soon.

**Comment by justfoolingaround** [comment]
Fixed as of `1.6.23`. Thanks for reporting this issue!
